### DISCUSSION TITLE
I am adding column pinnning in english view it is pinning correcty but in arabic view there is no column pinned for same table

### DISCUSSION THREAD

**Discussion by ixmaheshmaheshwari** [KevinVandy/material-react-table#1218] [Q&A]
I am adding column pinnning in english view it is pinning correcty but in arabic view there is no column pinned for same table

**Comment by KevinVandy** [comment]
Well are you using different IDs or something?

**Reply by ixmaheshmaheshwari** [reply]
no in both same id is there there to pinned but in english it is in right and in arabic it is in left  and the columns are pinned on both side and columns are different in en it is id in left and name in right and vice-versa
i tried with left=[langauage==='AR'?id:name] this is not working

**Comment by ixmaheshmaheshwari** [comment]
can anyone please answer these need it for urgent basis in my project
